### DISCUSSION TITLE
Add a "Retro experience" checkbox

### DISCUSSION THREAD

**Discussion by Iritscen** [Aleph-One-Marathon/alephone#257] [Ideas]
As AlephOne has increased in the complexity of the settings it offers and has begun providing high-res textures by default, it becomes harder and harder to get A1 back to the original settings.  Retrogamers and, I would assume, speedrunners, are looking for that 12/21/94 experience.  Please provide a checkbox somewhere prominent in Preferences which performs the needed adjustments to the graphics (e.g. textures, 3D perspective), controls (e.g. original mouse movement), UI (e.g. no quick save) and physics (e.g. restoring the bounce on downward-moving platforms) to get back to Original Marathon.

This checkbox would have to disable all the controls for the settings that it touches so that the user understands which settings can still be changed without losing the Retro setting.  Clicking (or maybe hovering over) such a disabled control could advise the user that they need to uncheck the Retro checkbox before they can change that setting, so it's clear why the control is grayed out.  The best place for the checkbox would probably be right at the bottom of the main Preferences screen, since it affects settings under multiple subscreens.

**Comment by ChristTrekker** [comment]
I second this. It would be great to set the relevant physics, graphics, and controls options (and whatever else) all at once.

Although I disagree with the implementation suggestion that Retro has to be unchecked before changes can be made. It would be nice to see all the things Retro implies clearly marked as such, perhaps by adding "(retro)" to the control description, yes. But I think it would be better to have the master Retro control be dimmed (but still toggleable) if any of the sub-settings was overridden individually, to indicate it's in a partially-retro state. Granted this makes is more complicated to write, but more convenient to players.

**Comment by tbcr** [comment]
I just want to mention, regardless of the fact that I do speedrun the game, that the mouse look was completely broken in Marathon 1 on mac, in general (Hopefully you are not intending on having mouse look intentionally broken if an option is set). Other than that, I do think this is an interesting idea.
